packages/xo-server/src/xo-mixins/proxies.mjs

```diff
@@ -57,6 +57,22 @@ function addProxyUrl(proxy) {
   proxy.url = url.href
 }
 
+async function addProxyVersion(proxy) {
+  try {
+    const versionInfo = await this.callProxyMethod(proxy.id, 'system.getServerVersion')
```

Comment:
Maybe don't call this method if the proxy is not running?
To avoid logging a "normal error"

Reply:
I think a non running proxy is not the expected case ( proxy are disposable tools that are destroyed if there is an issue) , and trying to detect if the proxy is running can lead to other errors that will have to be handled , especially in a flaky network